app/assets/javascripts/initializers/initializeAllFollowButts.js

```diff
@@ -1,10 +1,54 @@
 function initializeAllFollowButts() {
   var followButts = document.getElementsByClassName('follow-action-button');
   for (var i = 0; i < followButts.length; i++) {
-    initializeFollowButt(followButts[i]);
+    if (!followButts[i].className.includes("follow-user")) {
+      initializeFollowButt(followButts[i]);
+    };
   }
 }
 
+function fetchUserFollowStatuses(idButtonHash) {
+  var idString = "";
+  Object.keys(idButtonHash).forEach(function(id) {
+    idString += ("ids[]=" + id + "&");
+  });
+
+  fetch(`/follows/bulk_show?${idString}followable_type=User`, {
```

Comment:
this could be 

```js
const url = new URL("/follows/bulk_show", document.location);
const searchParams = new URLSearchParams();
Object.keys(idButtonHash).forEach((id) => {
  searchParams.append("ids[]", id);
});
searchParams.append("followable_type", "User");
url.search = searchParams;

fetch(url, { // etc
```

it's bit more lines but I think it's cleaner and uses less string interpolation.